wfe2/wfe_test.go

```diff
@@ -4462,3 +4469,80 @@ func TestCountNewOrderWithReplaces(t *testing.T) {
 	test.AssertEquals(t, responseWriter.Code, http.StatusCreated)
 	test.AssertMetricWithLabelsEquals(t, wfe.stats.ariReplacementOrders, prometheus.Labels{"isReplacement": "true", "limitsExempt": "true"}, 1)
 }
+
+func TestNewOrderRateLimits(t *testing.T) {
+	wfe, fc, signer := setupWFE(t)
+
+	features.Set(features.Config{UseKvLimitsForNewOrder: true})
+	defer features.Reset()
+
+	// Override the default ratelimits to only allow one new order per account
+	// per 24 hours.
+	txnBuilder, err := ratelimits.NewTransactionBuilder(ratelimits.LimitConfigs{
+		ratelimits.NewOrdersPerAccount.String(): &ratelimits.LimitConfig{
+			Burst:  1,
+			Count:  1,
+			Period: config.Duration{Duration: time.Hour * 24}},
+	})
+	test.AssertNotError(t, err, "making transaction composer")
+	wfe.txnBuilder = txnBuilder
+
+	var extantAKID []byte
+	for _, v := range wfe.issuerCertificates {
+		extantAKID = v.SubjectKeyId
+		break
+	}
```

Comment:
This is a neat trick to pick a random issuer to simulate for this certificate. Two notes:

 - Since it's a neat trick, it deserves a "why?" comment. E.g. "Pick a random issuer to 'issue' the extant certificate."
  - I think the intent would be clearer if you picked an _issuer_ here, and down on line 4502 referenced the SubjectKeyId. E.g.:
  
```
var issuer *issuance.Certificate
for _, v, := range wfe.issuerCertificates {
  issuer = v
  break
}
...
extantCert := &x509.Certificate{
  ...
  AuthorityKeyId: issuer.SubjectKeyId
```

Among other things, this avoids the naming awkwardness of `extantAKID`: what's an "extant AKID"? Oh it's the AKID associated with an extant certificate. But in the rewrite we can see that what we actually want is just to set the AKID to the SKID of some issuer.

(I notice this pattern is also used in TestCountNewOrderWithReplaces; it could probably be made clearer there, possibly as a followup)